roles/custom/matrix-bridge-hookshot/tasks/reset_encryptioon.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Resetting Hookshot's crypto store
+  ansible.builtin.command:
+    cmd: |
+      {{ devture_systemd_docker_base_host_command_docker }} run
+      --rm
+      --name={{ matrix_hookshot_container_url }}-reset-crypto
```

Comment:
It's quite unfortunate that the container-name (and what we call `_ident` in our roles) is called `matrix_hookshot_container_url` here, making you think it's `http://something-something` -- a value unsuitable for a container name.

In actuality, `matrix_hookshot_container_url` is just `matrix-hookshot` - a container name / identifier.